client/my-sites/domains/domain-search/index.tsx

```diff
@@ -200,7 +201,7 @@ class DomainSearch extends Component< DomainSearchProps > {
 			is_premium: isPremium,
 		} = suggestion;
 
-		this.props.recordAddDomainButtonClick( domain, 'domains', position, isPremium );
+		this.props.recordAddDomainButtonClick( domain, 'domains', position, isPremium, 'domains' );
```

Comment:
```suggestion
		this.props.recordAddDomainButtonClick( domain, 'domains', position, isPremium, 'domains/add' );
```
This code is only ran when adding a domain to an existing site, right? What do you think of naming the flow parameter here "domains/add" to not confuse this with the domain-only flow or the onboarding flow?